chapters/ch07.asciidoc

```diff
@@ -717,11 +717,11 @@ hypot(1, 2, 3)
 
 ==== 7.2.10 Bitwise Computation Helpers
 
-At the beginning of section 7.2, we talked about how some of the new +Math+ methods are specifically engineered towards making it easier to compile C into JavaScript. Those are the last three methods we'll cover, and they help us deal with 32-bit numbers.
+At the beginning of section 7.2, we talked about how some of the new +Math+ methods are specifically engineered for making it easier to compile C into JavaScript. Those are the last three methods we'll cover, and they help us deal with 32-bit numbers.
```

Comment:
Okay, it is back to "towards" - see commit below (1c1d289)... Oh, looks like I included my next edit in the reversal...